src/disposable.coffee

```diff
@@ -24,7 +24,9 @@ class Disposable
   dispose: ->
     unless @disposed
       @disposed = true
-      @disposalAction?()
```

Comment:
This ensured `@disposableAction` is only called if it is a function, the new code does not, can we preserve that old behavior?